Core.DomainModel/GDPR/Events/DataProcessingRegistrationRemovedFromItContractEvent.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Core.DomainModel.Events;
+using Core.DomainModel.Shared;
+
+namespace Core.DomainModel.GDPR.Events
+{
+    public class DataProcessingRegistrationRemovedFromItContractEvent : EntityLifeCycleEvent<DataProcessingRegistration>
+    {
+        public DataProcessingRegistrationRemovedFromItContractEvent(DataProcessingRegistration entity, ItContract.ItContract itContract) : base(LifeCycleEventType.Deleting, entity)
```

Comment:
the dpr is not getting deleted so this is an odd lifecycle event type.. also I don't think this is a lifecycle event in general. We use those for the lifecycle of the entities.. this is more specific. I will change it